examples/gateway-car/README.md

```diff
@@ -0,0 +1,32 @@
+# Gateway backed by a CAR File
+
+This is an example that shows how to build a Gateway backed by the contents of
+a CAR file. A [CAR file](https://ipld.io/specs/transport/car/) is a Content
+Addressable aRchive that contains blocks.
+
+## Build
+
+```bash
+> go build -o gateway
+```
+
+## Usage
+
+First of all, you will need some content stored as a CAR file. You can easily
+export your favorite website, or content, using:
+
+```
+ipfs dag export <CID> > data.car
+```
+
+Then, you can start the gateway with:
+
+
+```
+./gateway -c data.car -p 8040
+```
+
+Now you can access the gateway in [localhost:8040](http://localhost:8040). It will
```

Comment:
We either need to change this to `127.0.0.1` or add subdomain support on `localhost` (https://github.com/ipfs/go-libipfs/issues/149).

Website hosting will usually require subdomain support (otherwise static website generators have invalid relative paths, and no Origin isolation), so I'd include `localhost` subdomain gw in this example.